Sources/URLRequest+APIAdapter.swift

```diff
@@ -53,9 +53,19 @@ extension URLRequest {
     }
 
     private mutating func setURLEncoded(parameters: HTTPParameters) {
+        let allowedCharacters = CharacterSet(charactersIn: "!*'();:@&=+$,/?%#[] ").inverted
```

Comment:
Shouldn't be all non-ASCII characters encoded too?

```suggestion
        let allowedCharacters = CharacterSet(charactersIn: "!*'();:@&=+$,/?%#[]").inverted
```

Reply:
I don't know why, but non-ASCII characters (like £, ¢, ∞, §, etc.) are encoded too. 😮 ...And to your suggestion, I think the white-space should be encoded too. As not allowed characters I choosed URI reserved characters (by RFC 3986) and white-space.